core/sql/common/CharType.cpp

```diff
@@ -942,7 +942,8 @@ SQLVarChar::SQLVarChar(Lng32 maxLen,
 	       FALSE, allowSQLnull, isUpShifted, isCaseInsensitive,
 	       TRUE, cs, co, ce,
 	       encoding, vcIndLen),
-      clientDataType_(collHeap())  // Get heap from NABasicObject. Can't allocate on stack.
+      clientDataType_(collHeap()),  // Get heap from NABasicObject. Can't allocate on stack.
+      wasHiveString_(FALSE)
```

Comment:
I'm confused by the name. The string was a Hive string but it isn't any longer? Why should we care? (I imagine I'll find the answer later... there is some different semantic that we want downstream...)